features/search_companies.feature

```diff
@@ -136,11 +136,13 @@ Scenario: Search by company (and confirm admin can search with all company names
   And I am on the "all companies" page
   And I select "NoPayment" in select list t("activerecord.models.company.one")
   And I click on t("search")
+  Then I hide the companies search form
```

Comment:
If we add the `in the list of companies` qualifier to the "And I should <not> see...." statements, we can delete all of these `Then I show/hide the companies search form` lines